appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/StatelessSessionContainer.java

```diff
@@ -759,6 +772,8 @@ public void destroy(Object obj) {
 
     private static class PoolProperties {
         int maxPoolSize;
+        int maxWaitTimeInMillis = Integer.getInteger(String.format("fish.payara.ejb-container.%s", RuntimeTagNames.MAX_WAIT_TIME_IN_MILLIS),
```

Comment:
Have you created a new deployment descriptor field here? Does this need documenting by Mike's team.